docs/resources/index.md

```diff
@@ -3,8 +3,26 @@ title: Resources
 sidebar_position: 11
 ---
 
-# Resources
+- [Community](https://eosnetwork.com/community/)
+- [Blog](https://eosnetwork.com/blog/)
+- [Press Releases](https://eosnetwork.com/press/)
+- [Funding](https://eosnetwork.com/funding/)
+- [Developers Telegram](https://t.me/joinchat/EaEnSUPktgfoI-XPfMYtcQ)
+- [Stack Exchange](https://eosio.stackexchange.com/)
+- [Github EOS Foundation](https://github.com/eosnetworkfoundation)
+- [Github Antelope Frameowrk](https://github.com/antelopeio)
+- [Twitter](https://twitter.com/EOSnFoundation)
 
 ## Test Nets
 
-* Jungle https://jungletestnet.io/
+### EOS
+
+- [Jungle](https://jungletestnet.io/)
+- [CryptoKylin](https://www.cryptokylin.io/)
+
+### EOSIO
```

Comment:
I guess we want to list Antelope-based testnets here, not EOSIO? Else, we should not list them, I think.